postproengine/doc/README.md

```diff
@@ -6,10 +6,10 @@ The following workflows are available:
 - [avgplanes](avgplanes.md): Average netcdf sample planes
 - [correlate](correlate.md): Calculate the two-point correlation and integral lengthscale
 - [convert](convert.md): Converts netcdf sample planes to different file formats
+- [spod](spod.md): Compute SPOD eigenvectors and eigenvalues
```

Comment:
Hmm, that's interesting. I've only run it on linux but it does _not_ always show up in alphabetic ordering for me, as you point out. In any case, sorting the keys of the pluginlist before assigning to looptasks seems to enforce the desired behavior. I've committed that change to this PR

Reply:
Ah, I think the ordering is also determine by dependencies. Since the instantaneous executor import the spod executor for radial plots, the spod executor go moved up higher in the list. I'll need to open a different PR to fix this